CMakeLists.txt

```diff
@@ -530,6 +530,17 @@ if (ANDROID)
     add_custom_command (TARGET ${BOARD_CONTROLLER_NAME} POST_BUILD
         COMMAND "${CMAKE_COMMAND}" -E copy_if_different "${CMAKE_HOME_DIRECTORY}/compiled/${BOARD_CONTROLLER_COMPILED_NAME}" "${CMAKE_HOME_DIRECTORY}/tools/jniLibs/${ANDROID_ABI}/${BOARD_CONTROLLER_COMPILED_NAME}"
     )
+    foreach (DEPENDENCY_LIBRARY ${LIBFTDI_LIBRARIES})
```

Comment:
should it be under if (libftdi enabled)? and what is the content of this variable? Maybe its shorter wo loop?

Reply:
`LIBFTDI_LIBRARIES` is filled with all libraries libftdi needs linked, when libftdi is found.  It's empty (undefined values are empty) otherwise.  I did it in this format so that if there are ever other dependency libraries, they can be copied in simply by adding to the loop.  The loop should be kept in case libftdi changes what libraries it uses (like if libusb-1.0 becomes libusb-1.1).  Do you think there's value to wrapping it in an `if`?